src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -158,7 +158,7 @@ class NameInFilterSet(FilterSet):  # pylint: disable=E0602
             # pylint: disable=E0602
             name_in = CommaSeparatedValueFilter(name='name', lookup_type='in')
 
-            class Meta:
+            class Meta(object):
```

Comment:
If you need to fix code in a skipped test, can you instead please remove this test altogether?

Reply:
I will gladly remove it. Why were these tests added in the first place?